packages/wallets/src/features/cashier/flows/WalletDeposit/WalletDeposit.tsx

```diff
@@ -1,27 +1,31 @@
 import React from 'react';
 import { useActiveWalletAccount } from '@deriv/api-v2';
-import { CashierLocked, DepositCryptoModule, DepositFiatModule, DepositLocked } from '../../modules';
+import { CashierLocked, DepositCryptoModule, DepositFiatModule, DepositLocked, SystemMaintenance } from '../../modules';
 
 const WalletDeposit = () => {
     const { data } = useActiveWalletAccount();
     const isCrypto = data?.currency_config?.is_crypto;
 
     if (isCrypto) {
         return (
-            <CashierLocked>
-                <DepositLocked>
-                    <DepositCryptoModule />
-                </DepositLocked>
-            </CashierLocked>
+            <SystemMaintenance isDeposit>
```

Comment:
@lubega-deriv instead of having 2 props for deposit and withdrawal, can we have 1 prop (something like `transactionType` in TransactionStatus module which does the same thing)?

Reply:
Updated